src/customer.rs

```diff
@@ -10,9 +10,9 @@ pub use crate::cli::{customer as cli, customer::Cli};
 pub use crate::config::{customer as config, customer::Config};
 pub use crate::database::customer as database;
 pub use crate::defaults::customer as defaults;
-pub use crate::transport::client::{self as client, Chan, Client};
-pub use crate::transport::server::{self as server, Server};
 pub use crate::zkchannels::customer as zkchannels;
+pub use transport::client::{self as client, Chan, Client};
+pub use transport::server::{self as server, Server};
```

Comment:
Do these have to say `use X::self as X`? I thought if you keep the same name, you could just use `self` and it would automatically use the X as the name.

Reply:
apparently not 😇